.github/workflows/build.yml

```diff
@@ -161,7 +161,7 @@ jobs:
 
   deploy-preproduction:
     name: Deploy to Preproduction
-    needs: [push, deploy-development]
+    needs: [push]
```

Comment:
Is this a long term change or just while the infra is in flux?

Reply:
permanent if the team are happy with it. There's no overlap between envs